linchpin/defaults/schemas/schema_v4.json

```diff
@@ -709,6 +709,12 @@
                 "boot_dev": {
                     "type":"string"
                 },
+		"additional_storage": {
```

Comment:
When you update the schema, it seems that you don't convert tabs to spaces. If you are using vim, you can use `expandtab`. Here's a document on how to fix this as it causes inconsistencies in the schema documents at a minimum.

http://vim.wikia.com/wiki/Converting_tabs_to_spaces

Here's a screenshot of the problem in my vim editor.

![screenshot from 2017-11-10 15-18-55](https://user-images.githubusercontent.com/89334/32681075-e85be34e-c62a-11e7-861d-82a8175a7ef3.png)